src/App/component/SelectProject.jsx

```diff
@@ -42,7 +42,7 @@ function SelectProject({setCurrentProjectId}) {
   const jwtToken = localStorage.getItem("jwtToken")
 
   const loadProjects = () => {
-    Axios.get("http://localhost:9100/pvs-api/project/1",
+    Axios.get("http://localhost:9100/pvs-api/project/1/active",
```

Comment:
Curious about why we should always use `1` as the project ID in this request.

Reply:
Oh, this `1` is the member id, not project id.

Reply:
Maybe I can use `${memberId}`, if member id has bean passed to SelectProject page.

Reply:
They didn't pass the member id to frontend so here is still 1.

Reply:
If there's nothing determine the project ID, suggest remove the `1` in this RESTApi Path.

Reply:
It's member ID, backend service will use member ID to get the projects of the member.

Reply:
In this case, it is recommended not to put the constant in the API Path, but to define an id constant in the back-end service layer.

Reply:
So suggest change to 
```js
Axios.get("http://localhost:9100/pvs-api/project/active")
```
And 
```java
final String memberId = "1";
```
in our backend.

Reply:
Although we don't have registry functions, it doesn't mean that we won't have other members. If here need to refactor, I think we should let frontend get member ID instead of removing the argument.